spec/features/app_settings/registrations_enabled_spec.rb

```diff
@@ -2,25 +2,22 @@
 
 feature 'RegistrationsEnabled', devise: true do
   describe 'When registrations are disabled' do
-    xscenario 'user cannot create an account' do
+    scenario 'attempting to create new user redirects to root url' do
       allow(AppSetting).to(receive(:registrations_enabled?)).and_return(false)
 
-      visit root_path
+      visit new_user_registration_path
 
-      expect(page).to(have_selector("input[type=submit][value='Not accepting new accounts']"))
+      expect(page.current_url).to eq(root_url)
     end
   end
 
   describe 'When registrations are enabled' do
-    scenario 'user can create an account' do
+    scenario 'attempting to create new user does not dircect' do
       allow(AppSetting).to(receive(:registrations_enabled?)).and_return(true)
 
-      visit root_path
+      visit new_user_registration_path
 
-      expect(page).
-        to have_link(
-          t('upaya.links.sign_up'), href: new_user_start_path
-        )
+      expect(page.current_url).not_to eq(root_url)
```

Comment:
Rather than testing the negative expectation, what do you think about testing the specific expected URL? Otherwise, any URL besides the root url would allow this test to pass.